### ISSUE TITLE
Include Exception Message in Log of ZkBaseDataAccessor#get()

### ISSUE THREAD

**Issue by pkuwm** [apache/helix#1601]
### Description
Today, when I was debugging a zk serializer issue, I noticed that the thrown exception in 
```
ZkBaseDataAccessor#get(List<String> paths, List<Stat> stats, boolean[] needRead, boolean throwException)
```
doesn't contain the right error message.
Here is the current code:
```
catch (Exception e) {
  throw new HelixMetaDataAccessException(String.format("Fail to read nodes for %s", paths));
} finally {
```
Could you please include the error message extracted from the caught Exception in the newly thrown HelixMetaDataAccessException?

**Comment by pkuwm** [comment]
Will be fixed by #1609

**Comment by pkuwm** [comment]
fixed by #1609
